ddtrace/contrib/pytest/plugin.py

```diff
@@ -319,6 +319,14 @@ def pytest_runtest_protocol(item, nextitem):
     if pytest_module_item is not None and test_suite_span is None:
         test_suite_span = _start_test_suite_span(pytest_module_item)
 
+    if _CIVisibility.test_skipping_enabled() and _CIVisibility.should_skip(
+        item.name, test_suite_span.get_tag(test.SUITE), test_suite_span.get_tag(test.MODULE)
+    ):
+        # Replace test body by pytest skip call
+        item.obj = lambda **_: pytest.skip("Skipped by Datadog Intelligent Test Runner")
```

Comment:
nit: Instead of replacing the test body with pytest skip can we add pytest skip as a [marker](https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest.nodes.Node.add_marker)?

Reply:
Interesting idea! I'll experiment with it

Reply:
But in a follow-up PR ;)